### PR TITLE
Cloud Storage Backend.

### PR DESCRIPTION
**Pull request by Natim**
Replace https://github.com/mozilla-services/readinglist/pull/135

### CODE REVIEW

File: readinglist/storage/cloud_storage.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+        resp = self._client.get(url, params=params)
+        count = resp.headers.get('Total-Records')
+        records = resp.json()['items']
+        return records, count
+
+
+def load_from_config(config):
+    settings = config.registry.settings
+    server_url = settings.get('cloud_storage.host',
```
Comment (leplatrem) [inline readinglist/storage/cloud_storage.py]: ``cliquet.storage_url``  :)

**Comment by leplatrem** [comment]
At last, this was rebased and tests now pass :sweat:

**Comment by Natim** [comment]
LGTM

File: cliquet/storage/cloud_storage.py [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+
+from cliquet import logger
+from cliquet.storage import StorageBase, exceptions, Filter
+from cliquet.storage.memory import apply_sorting, get_unicity_rules
+from cliquet.utils import json, COMPARISON
+
+
+API_PREFIX = "/v0"
+
+DEFAULT_CLOUD_STORAGE_URL = 'https://cloud-storage.services.mozilla.com'
```
Comment (leplatrem) [inline cliquet/storage/cloud_storage.py]: XXX: rebase on master and move this to defaults settings

**Comment by leplatrem** [comment]
Shoudn't we rename it to kinto ?

**Comment by Natim** [comment]
> Shoudn't we rename it to kinto ?

I don't think we should.

**Pull request merged**